tests/vmi_lifecycle_test.go

```diff
@@ -165,7 +165,7 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			Eventually(logs,
 				2*time.Second,
 				500*time.Millisecond).
-				Should(And(ContainSubstring("internal error: Unable to get DBus system bus connection"), ContainSubstring(`"subcomponent":"libvirt"`)))
+				Should(And(ContainSubstring("At least one cgroup controller is required: No such device or address"), ContainSubstring(`"subcomponent":"libvirt"`)))
```

Comment:
@phoracek how does this work?

If I'm reading the code correctly, you're starting a VM and ensuring that succeeds, but afterwards you look at libvirt's logs and specifically look for an error message? Wouldn't that error have prevented libvirt (and thus the VM) from starting in the first place?

The change in the specifics of the error might be connected to internal changes in the order libvirt performs its checks, but I'm just uneasy that errors are raised in the first place.

Reply:
@andreabolognani  I was asking myself the same.

I think it just wants to see that libvirt has started, is running and printing logs. This was not the only error one I saw there...

This follows the patch that fixed this check for 5.6: https://github.com/kubevirt/kubevirt/commit/67e1d8b629737be30e254bac0aba68dcbfef00aa

Reply:
@phoracek looking at dc6b75cf31517f34c8c7b1aedb43139010a9a470, it seems like all three errors have been around for quite a long time, so I guess it's fine to look for any one of them.

I still wonder why some of them have stopped showing up over time? Order of initialization shouldn't affect this, since AFAICT you're just making sure that they eventually show up, so it's as if libvirt is emitting fewer errors over time. Not necessarily a bad thing :) but we should make sure that it's intentional and not masking some potential issue.

Another thing: `pkg/virt-launcher/virtwrap/util/libvirt_helper_test.go` contains what appears to be a fairly old libvirt log snippet, would it make sense to update it with a more recent one? Could be done with a separate PR of course.

Reply:
They seem terribly outdated. I found the formatted output here https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3268/pull-kubevirt-e2e-k8s-cnao-1.17/1261314930372513792, but I don't know how to get the basic one. This PR is blocking us from multiple tasks. The unit test is long outdated. I vote for a different PR.